<launch>
  <!-- main -->
	<node pkg="sq_scan_matching" name="icp_sq_scan_matcher" type="icp_sq_scan_matcher">
		<!-- <remap from="/cloud/lcl" to="/velodyne_points"/> -->
		<param name="pc_range" type="double" value="20.0"/>
		<param name="leafsize_source" type="double" value="0.15"/>
		<param name="leafsize_target" type="double" value="0.2"/>
		<param name="trans_epsilon" type="double" value="1e-3"/>
		<param name="stepsize" type="double" value="0.1"/>
		<param name="resolution" type="double" value="0.5"/>
		<param name="max_iterations" type="int" value="30"/>
		<param name="correspondence_distance" type="double" value="0.4"/>
	</node>
  <node pkg="rviz" type="rviz" name="rviz21" args="-d $(find sq_scan_matching)/config/icp.rviz" />
</launch>